src/cov.jl

```diff
@@ -64,6 +64,15 @@ cov
 
 
 """
+    cor(X, w::AbstractWeights; mean=nothing, vardim=1)
+
+Compute the Pearson correlation matrix of `X` along the dimension
+`vardim` with a weighting `w` .
+"""
+cor
```

Comment:
Instead of having the docstring atop the bare object here, you might as we put it above the method definition that it corresponds to below, since it's documenting a particular method.